### DISCUSSION TITLE
syntax error, unexpected '$url' (T_VARIABLE), expecting ')'

### DISCUSSION THREAD

**Discussion by meigieong** [spatie/laravel-webhook-client#97] [Q&A] [answered]
Hi i encountered this issue when i tried to use the webhook client, i've been googling for some time and i saw the answer was just to make sure we are using PHP 7.4, and i'm using PHP 7.4.

it happens on the file `vendor/spatie/laravel-webhook-client/src/WebhookClientServiceProvider.php` first is on the line 34 with `$url`, second is on the line 38, `$config`. I am using PHP 7.4.21 but still unable to get this through unless i change the `fn` to normal `function` . kindly advice.

![image](https://user-images.githubusercontent.com/79838625/126975442-53c18082-8e88-46ad-b690-f462e309e72c.png)

**Comment by meigieong** [comment]
hi may i know is this issue been resolved? :)

**Comment by meigieong** [comment]
hi spatie, appreciate if you could help to investigate what's wrong or advise please? thank you.

**Comment by freekmurze** [comment, marked as answer]
Our tests prove that the package doesn't contain a syntax error. Notice that the latest version of the package requires PHP 8.

**Comment by meigieong** [comment]
![image](https://user-images.githubusercontent.com/79838625/127983212-691b6eca-2594-4afb-a166-5f3709942e2c.png)
![image](https://user-images.githubusercontent.com/79838625/127983261-f9b0b641-909d-41d7-a83e-855358f4a2b4.png)
![image](https://user-images.githubusercontent.com/79838625/127983316-9be96f91-c8cd-4860-81bd-85977ebad8ee.png)

thank you for the advice. I'm using PHP 8 everywhere for this webhook package however it still has this error message. however, this syntax error doesn't affect the webhook process after I updated to use PHP 8.

**Reply by freekmurze** [reply]
Probably web requests are not handled via PHP 8: https://freek.dev/php-version
